src/onConsentChange.ts

```diff
@@ -79,7 +79,19 @@ const getConsentState: () => Promise<ComparedConsentState> = () => {
 
 			Promise.all([getTCDataPromise, getCustomVendorConsentsPromise])
 				.then((data) => {
-					const { consents } = (data[0] as TCFData).purpose;
+					const consents = {
```

Comment:
Why we are hardcoding 1-10 to false? Is it like their initial state or something? Is there a way to use the tcf response to populate their value?

Reply:
Ah think i got it, is it because purpose.consents keys will always be the same and thus override their default value, false?